pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -1521,6 +1521,12 @@ in
     '';
   };
 
+  kitty-scrollback-nvim = super.kitty-scrollback-nvim.overrideAttrs {
+    # Make it easier to add it to your Kitty config, e.g., using home-manager:
+    # programs.kitty.actionAliases."kitty_scrollback_nvim" = "kitten ${pkgs.vimPlugins.kitty-scrollback-nvim.kittenPath}";
+    passthru.kittenPath = "${super.kitty-scrollback-nvim}/python/kitty_scrollback_nvim.py";
+  };
+
```

Comment:
Not sure how I feel about this part. Without testing in place, this will break anytime upstream changes how they handle this internally. Looks to just be a small shortcut to save adding 20 characters.

Reply:
Ok, fair.